src/seeds.gratitude.cpp

```diff
@@ -46,21 +51,53 @@ ACTION gratitude::give (name from, name to, asset quantity, string memo) {
   update_stats(from, to, quantity);
 }
 
+ACTION gratitude::acknowledge (name from, name to, string memo) {
+  require_auth(from);
+
+  check( from != to, "gratitude: cannot give to self" );
+  check( is_account( to ), "gratitude: to account does not exist");
```

Comment:
Just a doubt, the "to" user doesn't have to be a seeds user?

Reply:
Done, added a check_user()